city_scrapers/spiders/cook_county.py

```diff
@@ -63,41 +77,18 @@ def _get_event_urls(self, response):
         event_urls = response.xpath('//div[@class="view-content"]//a/@href').extract()
         return list(set([x for x in event_urls if '/event/' in x and 'cancelled' not in x]))
 
-    def _parse_next(self, response):
-        """
-        Get next page. You must add logic to `next_url` and
-        return a scrapy request.
-        """
-        split_url = response.url.split('page=')
-        next_number = int(split_url[1]) + 1
-        if next_number <= 11:
-            return '{0}page={1}'.format(split_url[0], next_number)
-        else:
-            return None
-
-    def _parse_classification(self, response):
-        """
-        Parse or generate classification (e.g. town hall).
-        """
-        category = response.xpath('//div[@class="field event-categories"]/descendant-or-self::*/text()').extract()
-        category = ''.join(category).replace('Category:', '').strip()
-        if not category:
-            return 'Not classified'
-        else:
-            return category
-
-    def _parse_status(self, response):
-        """
-        Parse or generate status of meeting. Can be one of:
-
-        * cancelled
-        * tentative
-        * confirmed
-        * passed
-
-        By default, return "tentative"
-        """
-        return 'tentative'
+    @staticmethod
+    def _parse_classification(name):
```

Comment:
Can we classify JAC meetings as "Advisory Committee" instead of "Not classified"? JAC stands for Justice Advisory Council. Otherwise, looks good!

Reply:
Sure. Added.